src/plugins/intel_cpu/src/cpu_streams_calculation.cpp

```diff
@@ -17,6 +17,38 @@ using namespace InferenceEngine;
 namespace ov {
 namespace intel_cpu {
 
+std::vector<std::vector<int>> apply_core_type(const ov::hint::SchedulingCoreType input_type,
```

Comment:
Supposed this function will be integrated later, but my question is that if this function will generate a new copy of proc_type_table, whether there is any synchronized issue for multiple-threads scenario? Because totally the proc_type_table is a singleton data and shared for all streams or ov::core object.